backend/src/scheduler/projektiSchedulerService.ts

```diff
@@ -131,9 +177,9 @@ function formatScheduleDate(date: dayjs.Dayjs): string {
   return date.format("YYYY-MM-DDTHH:mm:ss");
 }
 
-function createScheduleParams(oid: string, date: dayjs.Dayjs): ScheduleParams {
+function createScheduleParams(oid: string, date: dayjs.Dayjs, eventType: ScheduledEventType): ScheduleParams {
   const dateString = formatScheduleDate(date);
-  return { oid, scheduleName: cleanScheduleName(`${createScheduleNamePrefix(oid)}-${dateString}`), dateString };
+  return { oid, scheduleName: cleanScheduleName(`${createScheduleNamePrefix(oid)}-${dateString}-${eventType}`), dateString };
```

Comment:
Taaksepäin yhteensopivuuden vuoksi pitäisikö jättää eventType pois import ja synchronize tyypeiltä. Tätä avainta kuitenkin käytetään vanhan ajastuksen poistamiseen. Toisaalta nyt tiedä onko niistä tuplista hirveästi haittaa. Lisäksi tuon nimen maksimipituus on 64 merkkiä, joten pitäisi käyttää enumimissa lyhempää string arvoa.